libraries/SPI/src/utility/xmc_spi_conf.h

```diff
@@ -60,6 +60,10 @@ extern XMC_SPI_t XMC_SPI_0;
 #define NUM_SPI  			1
 extern XMC_SPI_t XMC_SPI_0;
 
+#elif defined(XMC1400_Arduino_Kit)
+#define NUM_SPI  			1
+extern XMC_SPI_t XMC_SPI_0;
+
 #elif defined(XMC4400_Platform2GO)
```

Comment:
It feels all these sections of if else are including the same number of SPIs and same instance. Maybe they can be in one if block?

Reply:
Or use NUM_SPI defined in pins_arduino.h to reduce core modifications

Reply:
The implementation (C file) has a dedicated section for the XMC 1400 Kit for Arduino since most of the settings are unique to the board. Hence, a dedicated block in the H file to maintain similarity to the C file.

Reply:
Obviously you have a MISUNDERSTANDING about maintainability and readability.

The point about pins_arduino.h, is to SPECIFY the hardware specific definitions and flags THERE not in several other files.